src/test/regress/expected/multi_schema_support.out

```diff
@@ -1455,3 +1454,16 @@ DROP SCHEMA "CiTUS.TEEN2" CASCADE;
 NOTICE:  drop cascades to table "CiTUS.TEEN2"."CAPITAL_TABLE"
 DROP SCHEMA bar CASCADE;
 NOTICE:  drop cascades to table bar.test
+DROP SCHEMA test_schema_support CASCADE;
```

Comment:
Minor, it might be nice to have `SET client_min_messages TO WARNING` so that these notices are not printed